services/src/main/java/org/keycloak/credential/WebAuthnCredentialProvider.java

```diff
@@ -233,6 +240,27 @@ public boolean isValid(RealmModel realm, UserModel user, CredentialInput input)
         return false;
     }
 
+    protected WebAuthnAuthenticationManager getWebAuthnAuthenticationManager() {
+        WebAuthnPolicy policy = session.getContext().getRealm().getWebAuthnPolicy();
```

Comment:
Is it possible to make this working also with `WebAuthnPasswordlessCredentialProvider` ? Maybe it is sufficient to just extract WebAuthn policy retrieval in protected method and override in `WebAuthnPasswordlessCredentialProvider` with something like:

```
@Override
protected WebAuthnPolicy getWebAuthnPolicy() {
    return session.getContext().getRealm().getWebAuthnPolicyPasswordless();
}
```
?

Besides that, I see you already added the field to both "default" and "passwordless" policy, so hopefully that should be enough.

Reply:
Hi @mposolda, 

Just added, thanks!